cephcsi/Dockerfile

```diff
@@ -57,6 +57,8 @@ FROM updated_base
 LABEL org.opencontainers.image.source="https://github.com/cybozu/neco-containers"
 
 ARG SRC_DIR
+ARG CSI_IMAGE_VERSION
```

Comment:
IIUC, this value will be empty. `CSI_IMAGE_VERSION` should be defined and initialized before the first builder stage line `SRC_DIR` and `GO_ARCH`.